### ISSUE TITLE
New sort order: Oldest first

### ISSUE THREAD

**Issue by a1300** [jaegertracing/jaeger-ui#673]
Thank you for this wonderful project!

## Requirement - what kind of business use case are you trying to solve?

I would like to add a new sort order `Oldest first` to the drop down menu. If I have multiple (or repeating) errors I would like to see the first one to better analyze the incident.

![image](https://user-images.githubusercontent.com/11912239/103172837-a6566800-4856-11eb-8b61-7b7a37c4a5fe.png)

## Problem - what in Jaeger blocks you from solving the requirement?

No problem so far

## Any open questions to address

Do all backend storages support this option?

**Comment by yurishkuro** [comment]
This option is implemented purely on the UI side, it's not passed to the storage backend. So while it's fairly trivial to implement, it may not produce the results you are expecting. For example, I think Cassandra backend always reads the data latest first.

**Comment by a1300** [comment]
@yurishkuro does the `jaeger-ui` project already have some backend specific UI features (some things that only work with e.g. `Elasticsearch`) ?

**Comment by yurishkuro** [comment]
No, the UI is agnostic to the type of backend used.

**Comment by a1300** [comment]
@yurishkuro should I close this issue as `won't fix` ? Or are you open to include this Elasticsearch only issue? I found issue #268 which is specific to Elasticsearch (but not implemented yet).

How can I analyze a issue best without affecting my production system? I had multiple Elasticsearch crashes because I needed to increase the `limit` in order to see the **first** error span. Does an exporter / importer exists?
